test/glific/templates_test.exs

```diff
@@ -713,7 +713,7 @@ defmodule Glific.TemplatesTest do
                 "status" => "success",
                 "templates" => [
                   %{
-                    "category" => "TRANSACTIONAL",
```

Comment:
let's remove the hardcoding from here as well. We can access via template[0], template[1] etc